server/src/main/java/org/elasticsearch/action/admin/cluster/allocation/TransportGetAllocationStatsAction.java

```diff
@@ -68,7 +70,10 @@ public TransportGetAllocationStatsAction(
             TransportGetAllocationStatsAction.Response::new,
             threadPool.executor(ThreadPool.Names.MANAGEMENT)
```

Comment:
You wouldn't want to fork anymore with the deduplicator would you? Only fork in case you actually do the computation?

Reply:
Technically yes that's right of course. I'm always in two minds about adding more non-forking actions: they increase the risk of serious pain in future (for nontechnical reasons) at the cost of a little more latency right now. I'll let you have this one without prejudice tho 😉